pkg/kn/commands/route/list_test.go

```diff
@@ -113,26 +116,27 @@ func createMockRouteMeta(name string) *v1alpha1.Route {
 	return route
 }
 
-func createMockTrafficTarget(revision string, percent int) *v1alpha1.TrafficTarget {
+func createMockTrafficTarget(revision string, tag string, percent int) *v1alpha1.TrafficTarget {
```

Comment:
I think it would be better to separate the function for tagging to the traffic target like `createMockTrafficTargetWithTag`.

Reply:
That's good, but I'm afraid there will be too many duplicated code in this test file, as the test scenarios are very similar just with/without tag.